app/containers/BuyRam/index.js

```diff
@@ -34,8 +34,37 @@ const withConnect = connect(
 const withReducer = injectReducer({ key: 'BuyRam', reducer });
 const withSaga = injectSaga({ key: 'BuyRam', saga });
 
-export default compose(
+const enhance = compose(
   withReducer,
   withSaga,
-  withConnect
-)(BuyRamForm);
+  withConnect,
+  withStateHandlers(
+    {
+      unit: {
+        isEOS: true,
```

Comment:
Does the `isEOS` need to be nest within `unit` - can it not be flattened?

Reply:
You're correct it should! I started with something much more complicated earlier, like:
```
unit: {
   type: EOS|BYTES,
   label: ....
}
```
...but then I thought it was all an overkill. This is good enough for now, no early optimizations!

Reply:
You've changed!!!

Reply:
However I'd still like to nest the unit business logic using `mapProps`, in order to keep the props below clean. What do you think?

Reply:
Sounds good.